### ISSUE TITLE
Publish CPU load

### ISSUE THREAD

**Issue by chrisb86** [miaucl/linux2mqtt#12]
I think it would be nice to have sensors for the cpu load of the last 1, 5 and 15 minutes. It says much more about the total load of the system than the average cpu usage of the last 60 seconds.

**Comment by miaucl** [comment]
Hm, would have to redesign the collection process, but might be interesting indeed

**Comment by BNolet** [comment]
Considering that you can even go to the point of sending data every 1 second, I don't think it would be useful to have this package calculate averages on its own. It's meant to be used in tandem with MQTT to send the data there. Let the ingestor perform those calculations and let this package do what it's good at: sending snapshots of statistics

**Comment by miaucl** [comment]
I agree, which is why there is a default setting of `60s` which can be overwritten in the config and set to, lets say, `1s`. Like that, you can do with it afterwards whatever you like.

This is even described in the `Readme.md`.

> Instantaneous CPU utilization isn't all that informative. It's normal for a CPU to occasionally spike to 100% for a few moments and means that the chip is being utilized to its full potential. However, if the CPU stays pegged at/near 100% over a longer period of time, it is indicative of a bottleneck. The --cpu=60 parameter is the collection interval for the CPU metrics. Here CPU metrics are gathered for 60 seconds and then the average value is published to MQTT state topic for the sensor. A good value for this option is anywhere between 60 and 1800 seconds (1 to 15 minutes), depending on typical workloads.
